app/controllers/landing_page_controller.rb

```diff
@@ -19,12 +19,18 @@ class LandingPageController < ActionController::Metal
   # Adds helper_method
   include ActionController::Helpers
 
+  include FeatureFlagHelper
```

Comment:
This is a good change. You should also re-implement the way the topbar flag is checked based upon this so we avoid the extra DB call.